packages/http/src/mocker/negotiator/NegotiatorHelpers.ts

```diff
@@ -22,6 +23,30 @@ import {
 } from './InternalHelpers';
 import { IHttpNegotiationResult, NegotiatePartialOptions, NegotiationOptions } from './types';
 
+export const warnMsg = (contentTypes: string[]) => `Unable to find content for ${contentTypes}`;
```

Comment:
In general if we're not using this at least 2-3 times, let's refrain from creating another function.

Reply:
1st use: https://github.com/stoplightio/prism/pull/606/files#diff-47034315930357c592d473ec0fe19afcR179
  
2nd use:  https://github.com/stoplightio/prism/pull/606/files#diff-47034315930357c592d473ec0fe19afcR184